modulefiles/modulefile.ProdGSI.hera

```diff
@@ -8,7 +8,7 @@
 ##}
 #module-whatis " NOAA-EMC/GSI whatis description"
 
-setenv CRTM_FIX /scratch2/NCEPDEV/nwprod/NCEPLIBS/fix/crtm_v2.3.0
```

Comment:
@jderber-NOAA  Is this CRTM path change related to this work or an accidental inclusion?

Reply:
An accidental inclusion